draft-irtf-cfrg-vdaf.md

```diff
@@ -3480,6 +3544,21 @@ on the set of all aggregation parameters used for a Client's input share, and
 only proceed with the preparation and aggregation phases if that function call
 returns `True`.
 
+## Pseudorandom Generators and Random Oracles
+
+The objects we describe in {{prg}} share a common interface, which we have
+called Prg. However, these are not necessarily all modeled as cryptographic
+Pseudorandom Generators in the security analyses of our protocols. Instead, most
+of them are modeled as Random Oracles. For these use cases, we want to be
```

Comment:
* No need to capitalize this IMO (here and below)
* We should add a reference for a definition of "random oracle". BR93 ("Random Oracles are Practical") would be sufficient, or maybe there's a section in {{GKWW20}} we could point to?
```suggestion
of them are modeled as random oracles {{needs-ref}}. For these use cases, we want to be
```